<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 white, stop: 1 #eaeaea);
 border: 1px solid #cecece;
 border-radius: 10px;

}
QGroupBox {
 border: 1px solid #cecece;
border-radius: 10px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #4193ff, stop: 1 #9eeaff);
}
 QPushButton {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 white, stop: 1 #eaeaea);

 border: 0.5px solid rgb(206, 206, 206);
 }</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>481</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_swap">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DELA">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete after</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dCnt">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>45</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Count:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_aPos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Position:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_dPos">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Position:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_aAf">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add after</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to end</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_aOPos">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dOPos">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dPos">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>80</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ATD">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_aPos">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dCPos">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>118</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BACK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background-color: rgb(193, 193, 193);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #4193ff, stop: 1 #9eeaff);
}</string>
   </property>
   <property name="text">
    <string>Back to Main</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_view">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>View all data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IN">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>83</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_YP">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>149</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MD">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>116</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_LTY">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>182</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_IN">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>82</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory number:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MD">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>116</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_YP">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>150</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Year production:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_LTY">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>184</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Life time in years:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CLEAR">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>510</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CLEAR DATA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
